src/style.css

```diff
@@ -0,0 +1,93 @@
+html {
+  font-family: "Roboto", sans-serif;
+  font-size: 11px;
+  font-weight: 400;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+.item {
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 0 14px 15px;
+}
+
+.item__image {
+  display: flex;
+  width: 160px;
+  height: 134px;
+  margin: 30px auto 45px;
+}
+
+.item__rate,
+.item__price {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  flex-wrap: nowrap;
+  white-space: nowrap;
+}
+
+.item__description {
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 15px;
+  margin-bottom: 5px;
+}
+
+/* .item__number {
+} */
+
+.item__stars-block {
+  display: flex;
+  margin: 15px 0 14px;
+}
+
+.item__star-image {
+  background-image: url(images/star-active.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+  width: 15px;
+  height: 15px;
+}
+
+.item__star-image:nth-child(5) {
+  background-image: url(images/star.svg);
+}
```

Comment:
Use your stars block from the previous task, so that you could change 'item__stars-block' to 'item__stars-block--2' in your html and number of active stars would change.